package.json

```diff
@@ -94,7 +94,6 @@
     "express": "^4.21.2",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
-    "kerberos": "^2.2.1",
     "mocha": "^10.8.2",
```

Comment:
Kerberos fails to install on alpine because we don't support it.

Reply:
This was explicitly added in https://github.com/mongodb/node-mongodb-native/pull/4411 . I guess it wasn't needed?

Reply:
It looks like I forgot to update the lockfile.

Reply:
Sorry for the force push - I had to rebase with main.  But fixed.